frontend/components/WorkComponent.tsx

```
@@ -70,6 +71,12 @@ const WorkComponent = ({
               {work.status}
             </span>
           </div>
+          
+          <Link href={`/bidding/${work.id}`}>
+            <button className="w-full px-4 py-2 bg-slate-500 text-white rounded-md hover:bg-slate-600 transition-colors text-sm font-medium cursor-pointer">
+              View Details & Bid
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)